src/main/java/mate/controller/IndexController.java

```diff
@@ -1,9 +1,33 @@
 package mate.controller;
 
+import java.io.IOException;
+import java.time.LocalDateTime;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 import mate.dao.MyCoolResource;
 
+@WebServlet(urlPatterns = "/index")
 public class IndexController extends HttpServlet {
     private MyCoolResource myResource;
-    //TODO: implement
+
+    @Override
+    public void init() {
+        myResource = MyCoolResource.openResource();
+    }
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        myResource.write(LocalDateTime.now().toString());
+        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
+    }
+
+    @Override
+    public void destroy() {
+        myResource.close();
+        super.destroy();
```

Comment:
I think there's no  need to call super.destroy(), it has no implementation in a superclass
![image](https://user-images.githubusercontent.com/22300600/126166635-14a3cf9f-74f8-45ec-94a6-0fa20e3a315d.png)